lib/net/pool.js

```diff
@@ -2200,6 +2410,8 @@ class Pool extends EventEmitter {
         this.headerNext = node;
 
       this.headerChain.push(node);
+      if (this.options.neutrino)
+        await this._addBlock(peer, header, chainCommon.flags.VERIFY_NONE);
```

Comment:
`VERIFY_NONE` literally means verify nothing, that can't be safe! Lets look at this together